Leanplum-SDK/Classes/LPMessageTemplates.m

```diff
@@ -136,6 +137,33 @@
 static NSString *DEFAULTS_ASKED_TO_PUSH = @"__Leanplum_asked_to_push";
 static NSString *DEFAULTS_LEANPLUM_ENABLED_PUSH = @"__Leanplum_enabled_push";
 
+#pragma mark Helper View Class
+@interface LPHitView : UIView
+@property (strong, nonatomic) void (^callback)(void);
+@end
+
+@implementation LPHitView
+- (id)initWithCallback:(void (^)(void))callback
+{
+    if (self = [super init]) {
+        self.callback = [callback copy];
+    }
+    return self;
+}
+
+- (UIView *)hitTest:(CGPoint)point withEvent:(UIEvent *)event
+{
+    UIView *hitView = [super hitTest:point withEvent:event];
+    if (hitView == self) {
```

Comment:
shouldn't this be other way around ? If its not hit ?

Reply:
No this is correct. 
You read it like.... If it hits this view, call the callback and ignore it. Else continue traversing the view tree.
This essentially makes the view transparent.